src/ctelnet.h

```diff
@@ -41,6 +41,18 @@
 #include <queue>
 #include <string>
 
+#if defined (Q_OS_MACOS) || defined(Q_OS_LINUX)
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netinet/tcp.h>
+#endif
+
```

Comment:
Again, an `#elif defined(Q_OS_WIN32) is a better choice here IMO.

Reply:
Fixed.